src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -161,7 +162,7 @@ def forward(self, *inputs):
             *inputs: Input to the module.
 
         Returns:
-            torch.Tensor: Module output.
+            torch.Tensor: Independent distribution.
```

Comment:
please update this signature to include the actual inputs

Reply:
it appears that the return value is not a torch.Tensor then? it's a distribution?

Reply:
Yes the return value is `torch.distributions.independent.Independent` object